website/addons/mendeley/tests/test_models.py

```diff
@@ -11,7 +11,6 @@
 from website.addons.mendeley.tests.factories import (
     MendeleyAccountFactory,
     MendeleyUserSettingsFactory,
```

Comment:
`MendeleyUserSettingsFactory` is no longer being used in this file and can be removed